front/se_project/src/components/navBar/navLinks.jsx

```diff
@@ -82,10 +82,9 @@ const NavLinks = (props) => {
                         </ul>
                         </li>
                         <li className="nav-item mx-2">
-                        <Link className="nav-link" to="#">|</Link>
+                        <Link className="nav-link disabled" to="#">|</Link>
```

Comment:
I don't understand. why do you add this?

Reply:
disable to invariant the shape of mouse